app/assets/stylesheets/custom.css.scss

```diff
@@ -141,53 +135,54 @@ h1 {
   background-color: darkgrey;
 }
 
-// table td {
-//   font-family: monospace;
-//   border: 2px solid black;
-//   padding: 10px;
-//   align: center;
-// }
-//
-// .admin_orders_list {
-//   font-family: monospace;
-//   width: 65%;
-//   padding: 5px;
-//   border: 1px solid;
-//   height: 300px;
-//   overflow: auto;
-// }
-//
+table td {
+  font-family: monospace;
+  border: 2px solid black;
+  padding: 10px;
+  align: center;
+}
+
+.admin_orders_list {
+  font-family: monospace;
+  width: 65%;
+  padding: 5px;
+  border: 1px solid;
+  height: 500px;
+  overflow: auto;
+}
+
 .user_orders_list {
   font-family: monospace;
   width: 65%;
   padding: 5px;
   border: 1px solid;
-  height: 300px;
+  height: 500px;
   overflow: auto;
 }
-// .center_div {
-//   font-family: monospace;
-//   padding: 3px;
-//   margin-right: 10px;
-// }
-//
-// .dropdown-menu li:hover .sub-menu {
-//   font-family: monospace;
-//     visibility: visible;
-// }
-//
-// .dropdown:hover .dropdown-menu {
-//   font-family: monospace;
-//     display: block;
-// }
-//
-// .account{
-//     float:left
-// }
-//
+.center_div {
+  font-family: monospace;
+  padding: 3px;
+  margin-right: 10px;
+}
+
+.dropdown-menu li:hover .sub-menu {
+  font-family: monospace;
+    visibility: visible;
+}
+
+.dropdown:hover .dropdown-menu {
+  font-family: monospace;
+    display: block;
+}
+
+.account{
+    float:left
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space<br>Declaration should be terminated by a semicolon